subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -2051,20 +2051,17 @@ uint16_t ull_conn_lll_max_tx_octets_get(struct lll_conn *lll)
 
 /**
  * @brief Initialize pdu_data members that are read only in lower link layer.
+ *        Fields that are modified after static initialization are only
+ *        re-initialized.
  *
  * @param pdu_tx Pointer to pdu_data object to be initialized
  */
 void ull_pdu_data_init(struct pdu_data *pdu_tx)
 {
-	LL_ASSERT(pdu_tx);
-
-	pdu_tx->cp = false;
-	pdu_tx->rfu = 0U;
-#if !defined(CONFIG_SOC_OPENISA_RV32M1_RISCV32)
```

Comment:
Also this could be initialized once for all PDUs here instead of initialization in LLL. The field should be accessed in LLL for fragmented PDUs only (for fragments that follow first one).

Reply:
Same reply as to previous comment.